exercises/html-forms/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="index.html">
+      <label for="email">
+        <div>Email</div>
+      </label>
+
+      <input type="text" name="email" size="15" maxlength="30" />
+
+      <label for="select">
+        <div>Why is coding cool?</div>
+      </label>
+
+      <select name="Cool">
+        <option value="option">
+          Select an option
+        </option>
+
+        <option value="interesting">
+          Because it's interesting
+        </option>
+
+        <option value="fun">
+          Because it's fun
+        </option>
+
+        <option value="powerful">
+          Because it's powerful
+        </option>
+      </select>
+
+      <label for="language">
+        <div>Favorite Language (choose one)</div>
+      </label>
+
+      <input type="radio" name="language"
+        value="HTML" checked="checked" /> HTML
+
+      <input type="radio" name="language"
+        value="CSS" checked="checked" /> CSS
+
+      <input type="radio" name="language"
+        value="Javascript" checked="checked" /> Javascript
+
+      <label for="sign up">
+        <div>Would you like to sign up for more information?</div>
+      </label>
+
+      <input type="checkbox" name="sign up"
+        value="sign me up" checked="checked" /> Sign me up!
```

Comment:
The text here should be in a `label` element. Make sure you have a properly configured `for` attribute!